lib/Git/Hooks/CheckReference.pm

```diff
@@ -152,8 +152,13 @@ may configure it in a Git configuration file like this:
     acl = allow CRUD ^refs/tags/ by @cms
 
     # Users may maintain personal branches under user/<username>/
+    # All environmental variables are available. Just place them in single quotes.
```

Comment:
Single quotes? I didn't understand that.

Reply:
Changed. Just surround them with '{' and '}'.